app/models/user.rb

```diff
@@ -1,24 +1,28 @@
 class User < ActiveRecord::Base
-  has_many :article, dependent: :destroy
-  has_many :comment
-  belongs_to :role
+  has_many :articles, dependent: :destroy, inverse_of: :user
+  has_many :comment, inverse_of: :user
+  belongs_to :role, inverse_of: :users
+
+  ##### ALL VALIDATION
```

Comment:
whats this comment for? Is that validation added by devise or are you missing it here?